crates/bevy_render/src/renderer/mod.rs

```diff
@@ -104,23 +104,54 @@ pub fn render_system(world: &mut World, state: &mut SystemState<Query<Entity, Wi
     }
 }
 
+/// A wrapper to safely make `wgpu` types Send / Sync on web with atomics enabled.
+/// On web with atomics enabled the inner value can only be accessed on the
+/// `wgpu` thread or else a panic will occur.
```

Comment:
It should probably be noted that it will panic if dropped in another thread as well.

Reply:
@daxpedda Good call out. I updated the comment.